mmpose/datasets/datasets/base/kpt_2d_sview_rgb_vid_top_down_dataset.py

```diff
@@ -160,24 +169,6 @@ def _get_db(self):
     def evaluate(self, results, *args, **kwargs):
         """Evaluate keypoint results."""
 
-    @staticmethod
-    @abstractmethod
-    def _write_keypoint_results(keypoint_results, gt_folder, pred_folder):
-        """Write results into a json file."""
-
-    @abstractmethod
-    def _do_keypoint_eval(self, gt_folder, pred_folder):
-        """Keypoint evaluation.
-        Args:
-            gt_folder (str): The folder of the json files storing
-                ground truth keypoint annotations.
-            pred_folder (str): The folder of the json files storing
-                prediction results.
-
-        Returns:
-            List: Evaluation results for evaluation metric.
-        """
-
```

Comment:
@liqikai9 I removed these 2 abstract methods from the base class because they are neither used in the base class nor outside the dataset. If it's only used in the subclass, it should be directly implemented in the subclass. Please double-check if we can remove them.

Reply:
For now, these two methods are only used in the subclass so I think they can be removed. 

BTW, I suggest change the  name `_do_keypoint_eval `into `_do_python_keypoint_eval` in the subclass `TopDownPoseTrack18VideoDataset`, to be consistent with the similar methods in other datasets